enterprise/api/v1/views.py

```diff
@@ -308,21 +408,10 @@ def courses(self, request, enterprise_customer, pk=None):  # pylint: disable=inv
 
         Only courses with active course runs are returned. A course run is considered active if it is currently
         open for enrollment, or will open in the future.
-        ---
```

Comment:
This is another view that I question the need for after this work is done, but I am fairly certain that it is in use, so maybe we can figure out from @mattdrayer who is using this and come up with a way to migrate them down the line.